web-registry/src/pages/Marketplace/Storefront/Storefront.normalizer.ts

```diff
@@ -58,7 +58,9 @@ export const normalizeProjectsInfosByHandleMap = ({
 
     projectsMap.set(project?.id, {
       name: project.metadata?.['schema:name'] || project.id,
+      // TODO: why is this being called classIdName?
```

Comment:
if there's some credit class name, then we display it, else we just display the credit class id
maybe `classIdOrName` would be more explicit but sounds a bit long too

Reply:
i personally think that would be better, because when i saw this data in a downstream component, i forget where i was, the name definitely threw me off when i was inspecting the object.

Reply:
should we address this now if it's a small change? otherwise create a new issue?

Reply:
thanks!